packages/flutter/test/material/bottom_navigation_bar_theme_test.dart

```diff
@@ -0,0 +1,96 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:vector_math/vector_math_64.dart' show Vector3;
+
+
+void main() {
```

Comment:
There's a handful of tests that we've adopted for all component themes, see`ToggleButtonThemeTests`. Things like testing debug, hash, etc.

Reply:
Done